lib/travis/build/script/go.rb

```diff
@@ -64,14 +64,19 @@ def setup
         end
 
         def install
-          sh.if '-f Godeps/Godeps.json' do
-            sh.export 'GOPATH', '${TRAVIS_BUILD_DIR}/Godeps/_workspace:$GOPATH', retry: false
-            sh.export 'PATH', '${TRAVIS_BUILD_DIR}/Godeps/_workspace/bin:$PATH', retry: false
-
-            if go_version >= '1.1'
-              sh.if '! -d Godeps/_workspace/src' do
-                sh.cmd "#{go_get_cmd} github.com/tools/godep", echo: true, retry: true, timing: true, assert: true
-                sh.cmd 'godep restore', retry: true, timing: true, assert: true, echo: true
+          sh.if uses_15_vendoring do
+            sh.echo 'Using Go 1.5 Vendoring, not checking for Godeps'
+          end
+          sh.else do
+            sh .if '-f Godeps/Godeps.json' do
+              sh.export 'GOPATH', '${TRAVIS_BUILD_DIR}/Godeps/_workspace:$GOPATH', retry: false
```

Comment:
FWIW: There is no reason to do this if you issue a `godep restore` as `godep restore` checks out the revs recorded in Godeps.json and installs them into your $GOPATH. So either do this *OR* do a restore. Also `godep go <cmd>` does this for you.

PS: Hopefully this will all be moot at some point anyway since vendor.